From 40c0bfc2803daa86aa678c8e4b97a32f6da55161 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Petr=20P=C3=ADsa=C5=99?= <ppisar@redhat.com>
Date: Tue, 9 Jul 2019 14:07:27 +0200
Subject: [PATCH] Interpret outline.py by Python 3
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Fedora deprecated Python 2. The script seems working with both
Pythons. Because we need to choose a one, use Python 3.

https://bugzilla.redhat.com/show_bug.cgi?id=1676189
Signed-off-by: Petr Písař <ppisar@redhat.com>
---
 ftplugin/latex-suite/outline.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/ftplugin/latex-suite/outline.py b/ftplugin/latex-suite/outline.py
index ed72b34..475910d 100755
--- a/ftplugin/latex-suite/outline.py
+++ b/ftplugin/latex-suite/outline.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/python3
 
 # Part of Latex-Suite
 #
-- 
2.20.1

